### ISSUE TITLE
I want to get the csv file in /dataset/JSRT/jsrt_all.csv

### ISSUE THREAD

**Issue by bobiyaya** [Ikracs/medical_attack#1]
I'm glad you can provide the code of the paper, I am very interested in your work, can you provide the code in raw_data = .. /dataset/JSRT/jsrt_all.csv csv file, I would appreciate it.

**Comment by Ikracs** [comment]
> I'm glad you can provide the code of the paper, I am very interested in your work, can you provide the code in raw_data = .. /dataset/JSRT/jsrt_all.csv csv file, I would appreciate it.

Hi bro,
We have provided all the code needed. ‘jsrt_all.csv’ is a data file.  Please go to their website for downloading the data.

**Comment by bobiyaya** [comment]
Thanks for replying so quickly, I would like to ask where the weights/jsrt-unet/jsrt-unet_best.txt of the model is obtained

![image](https://github.com/user-attachments/assets/9c97c301-054a-41f7-a820-03969b21429d)

**Comment by Ikracs** [comment]
> Thanks for replying so quickly, I would like to ask where the weights/jsrt-unet/jsrt-unet_best.txt of the model is obtained
> 
> ![image](https://private-user-images.githubusercontent.com/87937989/369821212-9c97c301-054a-41f7-a820-03969b21429d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNDY0OTUsIm5iZiI6MTcyNzE0NjE5NSwicGF0aCI6Ii84NzkzNzk4OS8zNjk4MjEyMTItOWM5N2MzMDEtMDU0YS00MWY3LWE4MjAtMDM5NjliMjE0MjlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDAyNDk1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZDhmYjE3MzZiNTkwOGNjZWQ0NzIzYWRhZjc0YmJmNDdlYjdjOTEwZGI2MTYxNTI3ZTRmNDAwMzFjYTY1ZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.j32c_2YtKRhlnyUcrYHhhxpTI-ZFeawWeKtvhTo8g5k)

Sorry, I have no backup on my personal pc. You can train the model with PyMIC. It will take less than an hour.

**Comment by bobiyaya** [comment]
> > Thanks for replying so quickly, I would like to ask where the weights/jsrt-unet/jsrt-unet_best.txt of the model is obtained
> > ![image](https://private-user-images.githubusercontent.com/87937989/369821212-9c97c301-054a-41f7-a820-03969b21429d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNDY0OTUsIm5iZiI6MTcyNzE0NjE5NSwicGF0aCI6Ii84NzkzNzk4OS8zNjk4MjEyMTItOWM5N2MzMDEtMDU0YS00MWY3LWE4MjAtMDM5NjliMjE0MjlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDAyNDk1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZDhmYjE3MzZiNTkwOGNjZWQ0NzIzYWRhZjc0YmJmNDdlYjdjOTEwZGI2MTYxNTI3ZTRmNDAwMzFjYTY1ZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.j32c_2YtKRhlnyUcrYHhhxpTI-ZFeawWeKtvhTo8g5k)
> 
> Sorry, I have no backup on my personal pc. You can train the model with PyMIC. It will take less than an hour.

Thank you very much for your reply, I will try my best to make it happen

**Comment by bobiyaya** [comment]
Hello, I encountered an issue during the attack where the data dictionary does not contain the keys img_w, pix_w, cls_w, and softmax. Did you set these values in the jsrt_all.csv file?
![Uploading image.png…]()
